src/templates/page.jsx

```diff
@@ -80,26 +96,15 @@ Page.Content = PageContent;
 Page.DataCards = PageDataCards;
 Page.DataGrid = PageDataGrid;
 Page.DataGroups = PageDataGroups;
-Page.Details = PageDetailsWindow; // TODO: Deprecated. Alias name for Page.DetailsWindow. Remove in next major release.
+Page.DeprecatedFiltersDrawer = PageDeprecatedFiltersDrawer; // TODO: Deprecated. Remove in a major release.
+Page.Details = PageDetailsWindow; // TODO: Deprecated. Alias name for Page.DetailsWindow. Remove in a major release.
 Page.DetailsWindow = PageDetailsWindow;
-Page.FiltersDrawer = PageFiltersDrawer;
+Page.FiltersDrawer = PageFiltersDrawer; // TODO: Deprecated. Remove in a major release.
 Page.FiltersRail = PageFiltersRail;
-Page.Grid = PageDataCards; // TODO: Deprecated. Alias name for Page.DataCard. Remove in next major release.
-Page.Table = PageDataGrid; // TODO: Deprecated. Alias name for Page.DataGrid. Remove in next major release.
+Page.Grid = PageDataCards; // TODO: Deprecated. Alias name for Page.DataCard. Remove in a major release.
```

Comment:
Should it be `Page.DataCard` (singular) or `Page.DataCards` (plural)?

Reply:
`Page/Drawer.DataCards`. Plural because there can be more than one. It's my belief that `Page/Drawer.DataGrid` is plural as well, because "grid" can be referring to one or more.

Reply:
Is the actual alias correct then and just the comment is off?

Reply:
The comment is off. Sorry. :(